### PR TITLE
[RFR] Moar Doc

### PR DESCRIPTION
**Pull request by fzaninotto**
- [x] Writing your own field
- [x] Writing your own component (will need further love)
- [x] Import Tutorial images from CDN
- [x] Doc for `<ReferenceField>`
- [x] Doc for `<ReferenceManyField>`
- [x] Doc for `<DateInput>`
- [x] Doc for `<DisabledInput>`
- [x] Doc for `<RadioButtonGroupInput>`
- [x] Doc for `<ReferenceInput>`
- [x] Doc for `<SelectInput>`
- [x] Add anchors to doc chapters subheadings

### CODE REVIEW

**Comment by jpetitcolas** [comment]
Looks good so far. Perhaps can we already merge this first part?

**Comment by fzaninotto** [comment]
Yep, it can't harm.

**Comment by djhi** [comment]
Should we merge this then ?

**Comment by fzaninotto** [comment]
Switching to RFR

File: docs/Inputs.md [hunk 1]
```diff
@@ -92,6 +106,71 @@ import { LongTextInput } from 'admin-on-rest/mui';
...
+            { id: 'lifestyle', name: 'Lifestyle' },
+            { id: 'photography', name: 'Photography' },
+        ]} />
+    </Edit>
+);
+```
+
+![RadioButtonGroupInput](./img/radio-button-group-input.png)
+
+You can also customize the properties to use for the option name and value, thanks to the `optionText` and `optionValue` attributes:
```
Comment (djhi) [inline docs/Inputs.md]: s/option/options

File: docs/Inputs.md [hunk 2]
```diff
@@ -117,6 +196,49 @@ You can customize the rich text editor toolbar using the `toolbar` attribute, as
...
+    { _id: 123, full_name: 'Leo Tolstoi', sex: 'M' },
+    { _id: 456, full_name: 'Jane Austen', sex: 'F' },
+]} />
+```
+
+Lastly, use the `options` attribute if you want to override any of Material UI's `<SelectField>` attributes:
+
+{% raw %}
+```js
+<SelectInput source="category" options={{
```
Comment (djhi) [inline docs/Inputs.md]: Not related to this PR, but an `options` attribute on a a select might be confused with the select options elements

**Comment by djhi** [comment]
We're going to have an outstanding doc ! :+1:

**Pull request merged**